plugins/inputs/nginx_plus/nginx_plus.go

```diff
@@ -0,0 +1,607 @@
+package nginx_plus
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal/errchan"
```

Comment:
I was working off an old master, so I will need to rebase and get rid of all calls to errchan.

Reply:
:+1: